railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -37,3 +37,9 @@
 
 # Set both the `:open_timeout` and `:read_timeout` values for `:smtp` delivery method.
 # Rails.application.config.action_mailer.smtp_timeout = 5
+
+# The ActiveStorage video previewer will now use scene change detection to generate
+# better preview images (rather than the previous default of using the first frame
+# of the video). To keep the previous default use:
+# `Rails.application.config.active_storage.video_preview_arguments =
+#   ["-y", "-vframes", "1", "-f", "image2", "-"]`
```

Comment:
These are meant to be the new defaults, not the old ones. This [file only exists in upgraded apps][1] which have `load_defaults "6.1"` and you then uncomment each one until you're happy. 

[1]: https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/railties/lib/rails/generators/rails/app/app_generator.rb#L557-L561

Reply:
Understood :) I've never used this method for upgrading so I missed the point of the file :D